NMS-14913: Enlink daemon doc restructure (#5416)
OpenNMS/opennms: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc

= Enlinkd

The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.adoc[topology information] via SNMP to determine neighboring nodes.
The daemon essentially asks each device the following question: "What is the network topology from your point of view?"
The enlinkd discovery process attempts to discover bridge domain links with the data coming from all collected bridge forwarding tables.
The enlinkd updaters correlate all collected topology data to generate a global topology layout of your network.

> .Enlink configuration file overview

Reviewer: ```suggestion
.Enlinkd configuration file overview
```

| enlinkd.log
| Logs relating to enlinkd server events.
|===

== Enlinkd configuration

If multiple protocols are enabled, links will be discovered for each enabled discovery protocol.
The topology web UI visualizes links for each discovery protocol.
For example, if you start CDP and LLDP discovery, the web UI visualizes a CDP link and an LLDP link.

> Restart {page-component-title} or just the enlinkd daemon to apply configuration changes.

Reviewer: The 
enlinkd configuration file over table says restart is not required, but this text indicates the opposite.

Author: This is a wording issue that plagues all the daemon files. A restart of the full horizon/Meridian server isn't required to apply changes, just a reload of the daemon.

Reviewer: We can consider rephrasing if we go through the section again. It's fine for now.

.enlinkd-configuration parameters
[options="header"]
[cols="1,3,1"]
|===
| Attribute
| Description
| Default

3+| *Integer*

| threads

> | Number of parallel threads collectors, updaters, and discovery use.

Reviewer: ```suggestion
| Number of parallel thread collectors, updaters, and discovery use.
```
"threads" sounds wrong to me, but if it's common use, ignore my comment.

Author: It's the number of threads used by collectors, updaters, and discovery. I'll tweak the wording to clarify.

Author: ```suggestion
| Number of parallel threads for enlinkd collector, updater, and discovery tasks.
```
